app/security/Roles.scala

```diff
@@ -182,7 +182,7 @@ object Roles {
 
   val userJourneySequence: List[(CsrAuthorization, Call)] = List(
     ApplicationStartRole -> routes.HomeController.present,
-    PersonalDetailsRole -> routes.FastStreamApplication.generalDetails(None),
+    PersonalDetailsRole -> routes.PersonalDetailsController.present(None),
```

Comment:
I think roles is not a good name in this context, on top of that now we have two set of roles: Admin, ServiceAdmin, etc and these other roles. Is there a better name for the last ones? Maybe "Permission"?